dev/select_constraints.h

```diff
@@ -270,30 +270,30 @@ namespace sqlite_orm {
 
         template<typename Moniker, class ExplicitCols>
         struct cte_builder {
-            ExplicitCols explicitColumns;
+            ExplicitCols _explicitColumns;
```

Comment:
do you want to replace all `this->` places with `_` prefix?

Reply:
Do you mean across the entire library? That would be especially good for the internally created classes with public members. I would save that for another day, is that okay?

Reply:
Yes I mean across the entire lib. I am asking in terms of your global plans actually cause I am a fan if `this->` and removing it triggers me a bit. On another hand I am ok with the leading underscore. But combining both looks ugly so I am just asking.

> I would save that for another day, is that okay?

Not a problem at all.

Reply:
I am not a fan of `this->` at all, it makes things much harder to read and is usually not necessary from a syntax perspective :)
But I am OK with either style.

So we should get together one day and find a guideline for the coding style that everyone follows.
By the way, I was not the one who introduced the leading underscore. For structs as public interfaces (even if they are from the internal namespace) the leading underscore is important, as you already mentioned. Firstly because of sorting in an IDE, and secondly to easily distinguish between public and private members.